pycrdt_websocket/yroom.py

```diff
@@ -304,3 +306,33 @@ async def serve(self, websocket: Websocket):
                 self.clients.remove(websocket)
         except Exception as exception:
             self._handle_exception(exception)
+
+    def local_update_awareness(self, type: str, changes: tuple[dict[str, Any], Any]) -> None:
+        """
+        Callback to broadcast the server awareness to clients.
+        """
+        if not changes[1] == "local":
+            return
+
+        if self._task_group is not None:
+            updated_clients = [
+                *changes[0].get("added", []),
+                *changes[0].get("filtered_updated", []),
+            ]
```

Comment:
The changes consist of `"added"`, `"updated"` and `"removed"` entries only, not `"filtered_updates"`. Should we include `"removed"`?
```suggestion
            updated_clients = [v for value in changes[0].values() for v in value]
```

Reply:
Also I added a test to broadcast only if there is an effective change (not 'update').